sdk/avs/Azure.ResourceManager.Avs/CHANGELOG.md

```diff
@@ -1,14 +1,10 @@
 # Release History
 
-## 1.6.0-beta.1 (Unreleased)
+## 1.6.0 (2025-12-18)
 
 ### Features Added
 
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
```

Comment:
It would be great if the list of changes could be generated similar to the other SDKs.